frameworks/Lua/lapis/setup.sh

```diff
@@ -4,6 +4,8 @@ sed -i 's|DBHOSTNAME|'"${DBHOST}"'|g' config.lua
 sed -i 's|DBHOSTNAME|'"${DBHOST}"'|g' config.moon
 sed -i 's|DBHOSTNAME|'"${DBHOST}"'|g' nginx.conf
 
+export OPENRESTY_HOME=${IROOT}/openresty-1.7.7.1
+export LAPIS_OPENRESTY=${IROOT}/openresty-1.7.7.1
```

Comment:
Does this need to be set? It's the same value as `$OPENRESTY_HOME` - if it's required, please add a comment as to why it's the same value.